python/setup.py

```diff
@@ -22,6 +22,7 @@
               sources=cython_files,
               include_dirs=[
                 '../cpp/include/cudf',
+                '../cpp/include',
```

Comment:
Is there a reason this change was needed? everything should live in the `cudf` folder in include, no?

Reply:
`file_2.h` includes `file_1.h`. `file_2.h` moved down a level to `cpp/include/cudf/legacy/file_2.h`. So now it has the line `#include <cudf/file_1.h>` instead of `#include "file_1.h"`. `include/cudf/legacy/file_2.h`. So with only `'../cpp/include/cudf'` in `setup.py` it would try to find `'../cpp/include/cudf/cudf/file.h'`